.travis.yml

```diff
@@ -18,12 +18,12 @@ install:
     - conda install --yes --quiet numpy=1.13.1 scipy=0.19.1 matplotlib pandas=0.20.3
     - pip install flake8 pytest nose coverage coveralls
     - |
-      git clone --depth 1 https://github.com/mne-tools/mne-python.git -b maint/0.16
+      git clone --depth 1 https://github.com/mne-tools/mne-python.git
```

Comment:
@sappelhoff I forget why we had to use the `master` branch here. This is not good ...

I can live with `master` branch for bids-validator but not for `mne`